dev-lang/erlang/files/epmd.init-r1

```diff
@@ -0,0 +1,26 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the Erlang Public License 1.1
+
+ZT_ADDRESS=""
+pidfile="/var/run/epmd.pid"
+command_args="${command_args}"
```

Comment:
This is redundant.

Reply:
Also, you can remove ZT_ADDRESS=""

Other than that it looks fine (also what mgorny said)

Reply:
fix